src/components/GRTFeedbackView/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { DateTime } from 'luxon';
+// eslint-disable-next-line camelcase
+import { GetGRTFeedback_grtFeedbacks } from 'queries/types/GetGRTFeedback';
+
+import HelpText from 'components/shared/HelpText';
+
+type GRTFeedbackViewProps = {
+  // eslint-disable-next-line camelcase
+  grtFeedbacks: GetGRTFeedback_grtFeedbacks[];
+};
+
+const GRTFeedbackView = ({ grtFeedbacks }: GRTFeedbackViewProps) => {
+  const { t } = useTranslation('businessCase');
+
+  return (
+    <>
+      <h2 className="margin-bottom-3 margin-top-0">
+        {t('grtFeedback.header')}
+      </h2>
+      {grtFeedbacks.some(grtFeedback => grtFeedback.feedbackType === 'GRB') && (
+        <div>
+          <h3 className="margin-bottom-1">{t('grtFeedback.grbSubhead')}</h3>
+          <HelpText className="margin-bottom-2">
+            {t('grtFeedback.grbHelpText')}
+          </HelpText>
+          {grtFeedbacks
+            .filter(grtFeedback => grtFeedback.feedbackType === 'GRB')
+            .sort(
+              (a, b) =>
+                DateTime.fromISO(a.createdAt).toMillis() -
+                DateTime.fromISO(b.createdAt).toMillis()
+            )
+            .map(grtFeedback => {
+              return (
+                <div>
```

Comment:
It appears to me that the GRB and Business Owner feedbacks share the same markup. Is it possible to reuse this JSX?